scrapers/SexLikeReal.yml

```diff
@@ -0,0 +1,29 @@
+name: "SexLikeReal"
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - sexlikereal.com
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    common:
+      $info: //div[@class="u-py--four desktop:u-ml--twn-one u-relative"]/div
+    scene:
+      Title: //div[@class="u-relative u-z--zero"]//h1/text()
+      Date:
+        selector: $info//time/@datetime
+        parseDate: 2006-01-02
+      Details: $info/div[4]/div[2]/text()
```

Comment:
in some cases this is not enough https://www.sexlikereal.com/scenes/three-cheers-for-threesomes-13355
also no need to use positions since class seems unique
```
Details:
        selector: //div[@class="u-mb--four u-lh--opt u-fs--fo u-fw--medium u-lw"]//text()
        concat: " "
```